RecoLocalCalo/HcalRecAlgos/src/HcalDeterministicFit.cc

```diff
@@ -97,7 +98,9 @@ void HcalDeterministicFit::getLandauFrac(float fC, int offset, double fpar0, dou
 
 void HcalDeterministicFit::phase1Apply(const HBHEChannelInfo& channelData,
 				       float& reconstructedEnergy,
-				       float& reconstructedTime) const
+				       float& reconstructedTime,
+                       float& previousEnergy,
+                       float& nextEnergy) const
```

Comment:
@thongonary 
This modification require to changes the phase1Apply inside the HcalDeterministicFit.h and inside the modules that call them.